Assets/AnkrDemo/Scripts/WebGLHeaderWalletsPanel.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using AnkrSDK.Data;
+using AnkrSDK.Examples.UseCases.WebGlLogin;
+using Demo.Scripts;
+using Newtonsoft.Json;
+using UnityEngine;
+
+namespace AnkrDemo.Scripts
+{
+	public class WebGLHeaderWalletsPanel : MonoBehaviour
+	{
+		[SerializeField]
+		private GameObject _panel;
+		[SerializeField]
+		private GameObject _buttonPrefab;
+		[SerializeField]
+		private WalletItem[] _wallets;
+		private List<HeaderWalletButton> _buttons;
+		public Action<Wallet> ConnectTo;
+
+		private void Start()
+		{
+			_buttons = new List<HeaderWalletButton>();
+			CreateButtons();
+		}
+
+		private void CreateButtons()
+		{
+			foreach (var wallet in _wallets)
+			{
+				var button = Instantiate(_buttonPrefab);
```

Comment:
change _buttonPrefab type to HeaderWalletButton and you will not have to use .GetComponent()
also this way you can ensure that the prefab is the correct one

Reply:
According with your recomendation I will instantiate just script for this buttons. But I need buttons.

Reply:
it does not work like that. You will link a prefab into _buttonPrefab field and it will instantiate it. But instead of returning a gameobject it will return a script instance on it.